src/plugins/intel_cpu/src/utils/print_model.hpp

```diff
@@ -0,0 +1,424 @@
+                } else {
+                    if (literal_consts.count(iop))
+                        os << sep << tag << literal_consts[iop];
+                    else
+                        os << sep << tag << opname[iop.get()];
+                }
+                sep = ", ";
+            }
+            os << "}";
+
+            // attributes as AnyMap
+            std::stringstream ss2;
+            OstreamAttributeVisitor osvis(ss2);
+            op->visit_attributes(osvis);
+            auto str_attr = ss2.str();
+            if (str_attr.size())
+                os << ", {" << str_attr << "}";
+            os << ");   //  tensor_array<" << get_output_values_info(op) << "> " << op->get_friendly_name();
+
+            os << "(";
+            sep = "";
+            for (size_t i = 0; i < op->get_input_size(); i++) {
+                auto vout = op->get_input_source_output(i);
+                auto iop = vout.get_node_shared_ptr();
+                os << sep << tag << iop->get_friendly_name();
+                if (iop->get_output_size() > 1) {
+                    auto out_port = vout.get_index();
+                    os << "[" << out_port << "]";
+                }
+                sep = ", ";
+            }
+            os << ")" << std::endl;
+
+            //==========================================================
+            // auto op_output_size = op->get_output_size();
+            // if (op_output_size > 1) {
+            //     os << "    " << name << "->set_output_size(" << op_output_size << ");" << std::endl;
+            // }
+        }
+
+        // recursively output subgraphs
+        if (auto msubgraph = std::dynamic_pointer_cast<op::util::MultiSubGraphOp>(op)) {
+            auto cnt = msubgraph->get_internal_subgraphs_size();
+            for (size_t i = 0; i < cnt; i++) {
+                os << "    MultiSubGraphOp " << tag << msubgraph->get_friendly_name() << "[" << i << "]" << std::endl;
+                dump_cpp_style(os, msubgraph->get_function(i));
+            }
+        }
+    }
+    os << prefix << "}\n";
+}
+
+}  // namespace detail
+
+class OPENVINO_API PrintModel : public ov::pass::ModelPass {
+public:
+    OPENVINO_RTTI("ov::pass::PrintModel");
+
+    PrintModel(std::string file_name) {
+        static int dump_index = 0;
+        m_file_name = std::string("modelprint_") + std::to_string(dump_index) + "_" + file_name;
+        dump_index++;
+    }
+    ~PrintModel() {}
+
+    bool run_on_model(const std::shared_ptr<ov::Model>& model) override {
+        if (m_file_name.empty())
+            return false;
+
+        std::cout << "PrintModel to " << m_file_name << " ... " << std::flush;
+        std::ofstream ofs(m_file_name);
+        if (!ofs) {
+            // OPENVINO_WARN << "Error opening file " << m_file_name << " for output" << std::endl;
+            return false;
+        }
+        detail::dump_cpp_style(ofs, model);
+        ofs.close();
+        std::cout << "done." << std::endl;
```

Comment:
Same

Reply:
removed

Reply:
Still there

Reply:
Sorry, I forgot commit the file, will remove